docs/deploy.md

```diff
@@ -17,17 +17,19 @@
 #
 -->
 
-# Deploy manually
+# Deploy Manually
 
-## Clone the project
+## Clone The Project
 
 ```sh
 $ git clone -b v2.0 https://github.com/apache/apisix-dashboard.git
 
 $ cd apisix-dashboard
 ```
 
-## Build the manager-api
+## Build
+
+### Manager-api Dependencies
```

Comment:
Requirements for manager-api

Reply:
emm...use `dependencies` better, keep it uniform with Apache APISIX